lldb/source/Plugins/Language/Swift/SwiftRuntimeFailureRecognizer.cpp

```diff
@@ -67,23 +74,55 @@ lldb::RecognizedStackFrameSP SwiftRuntimeFailureFrameRecognizer::RecognizeFrame(
                                                   runtime_error));
 }
 
-lldb::StackFrameSP
-SwiftRuntimeFailureRecognizedStackFrame::GetMostRelevantFrame() {
-  return m_most_relevant_frame;
+lldb::RecognizedStackFrameSP
+SwiftRuntimeInstrumentedFrameRecognizer::RecognizeFrame(
+    lldb::StackFrameSP frame_sp) {
+  if (frame_sp->GetFrameIndex())
+    return {};
+
+  ThreadSP thread_sp = frame_sp->GetThread();
+  if (!thread_sp)
+    return {};
+  StackFrameSP most_relevant_frame_sp = thread_sp->GetStackFrameAtIndex(5);
```

Comment:
Can you elaborate on why we get the 5th frame?

Reply:
Great point. I should probably change this to go up until we leave the standard library instead of hardcoding 5 frames.